tools/webpack-make

```diff
@@ -100,38 +100,35 @@ function generateDeps(makefile, stats) {
     if (dir.startsWith(cwd + '/'))
         dir = dir.substr(cwd.length + 1);
 
-    for(asset in stats.compilation.assets) {
+    const uncompressed_patterns = [
+        '/manifest.json$', '/override.json$',
+        '^dist/static/', // cockpit-ws cannot currently serve compressed login page
+        '^dist/shell/index.html$',  // COMPAT: Support older cockpit-ws binaries. See #14673
+        '\.png$', '\.woff$', '\.woff2$', '\.gif$'
+    ].map((r) => new RegExp(r));
+
+    for (asset in stats.compilation.assets) {
         output = path.join(dir, asset);
         fs.utimesSync(output, now, now);
 
         if (!output.endsWith("/manifest.json") && !output.endsWith(".map"))
             outputs[output] = output;
 
-	if (output.indexOf("/test-") !== -1 && output.endsWith(".html")) {
-            tests[output] = output;
+        if (output.includes("/test-")) {
+            if (output.endsWith(".html")) {
+                tests[output] = output;
+            }
             continue;
```

Comment:
This is a logic change. Now it continues for any test-* file which does not end with .html. I did not think about whether the old or new version is correct.

Reply:
It `continue;`s for all `test-*` files.  That's intended: everything below that point is about dealing with install paths.

This really is a straight refactor: there are no changes to the output here.

Reply:
(I was so careful about testing it, because changing this logic has burned us in the past).